Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml

```diff
@@ -0,0 +1,71 @@
+category: Utilities
```

Comment:
are you sure this is the right category for this integration ?

Reply:
I think we can change it to "Authentication".